app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.spec.tsx

```diff
@@ -6,7 +6,7 @@ import { rest } from 'msw';
 import type { SetupServer } from 'msw/node';
 import { SWRConfig } from 'swr';
 import { I18nContext } from '@18f/identity-react-i18n';
-import { ADDRESS_SEARCH_URL, LOCATIONS_URL } from '@18f/identity-address-search';
+import { ADDRESS_SEARCH_URL, LOCATIONS_URL } from '@18f/identity-address-search/source';
```

Comment:
wouldn't we want to run specs against the original file? not the generated one?

Reply:
Ideally neither would have to dig into paths like that 😕 

In one of the Slack conversations, I [mentioned](https://gsa-tts.slack.com/archives/C0NGESUN5/p1686675441186639?thread_ts=1686673764.810239&cid=C0NGESUN5) the [`resolve.conditionNames`](https://webpack.js.org/configuration/resolve/#resolveconditionnames) as a possibility, so that the actual imports could be the bare package name, while still internally resolving to some "source" path. Were you able to give that a shot?